dev-packages/node-integration-tests/suites/tracing/tedious/scenario.js

```diff
@@ -10,7 +10,7 @@ Sentry.init({
 
 const { Connection, Request } = require('tedious');
 
-const config = {
+const connection = new Connection({
```

Comment:
I changed this because I get TS autocomplete when passing the object in directly (note that this is a js file, so importing types is annoying).